lib/src/code_field/code_field.dart

```diff
@@ -510,6 +534,38 @@ class _CodeFieldState extends State<CodeField> {
     _suggestionsPopup!.markNeedsBuild();
   }
 
+  void _onSearchSettingsChanged() {
```

Comment:
Follow the format of `_onPopupStateChanged`: Test if should show. If not, dismiss, return.